### PR TITLE
added gitpod

### PR DESCRIPTION
**Pull request by pollend**
added .gitpod.yml file and tag to launch movingblocks website in gitpod.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+tasks:
+  - init: >
+    gem install bundler jekyll
+    bundle install
```
Comment (NicholasBatesNZ) [inline .gitpod.yml]: I'm pretty sure that package installation should be in the Dockerfile

Comment (pollend) [reply .gitpod.yml]: anyways I screwed up the configuration.

**Comment by pollend** [comment]
just verified and this works correctly

**Comment by pollend** [comment]
I was doing some testing it's faster to use the pre-built image then build a docker image. It lets you get to the workspace quicker then waiting on a docker image to be built then run.

**Pull request closed without merging**